website/guides/sgb_border.md

```diff
@@ -0,0 +1,175 @@
+# Adding an SGB border
+
+This document was written to aid developers of DMG compatibile homebrew with adding Super Game Boy borders
+
+I cannot guarantee skipping certain parts won't leave you confused, so if you wish to add SGB borders to already SGB capable software (that lacks borders), try not to skip anything
+
+## Enabling SGB features
+
+Before we can do anything, we must first specify in the header that this game is an SGB game.
+
+To enable SGB features: <!-- i think this is worded a little awkwardly, but i cant exactly think of how to fix it >< -->
```

Comment:
I wanted to mention that what controls the "enabling of SGB features" is the ROM's header, and not e.g. code. I figured that it could help with troubleshooting?

Reply:
Uh, okay, I see your point.. We should then try to simplify it a bit (e.g. to help restrict the cases in which really this "troubleshooting" may happen and the user should be aware of the fact it may involve SGB side, if I understood you correctly)